test/functional/test_runner.py

```diff
@@ -503,10 +513,11 @@ def check_script_list(src_dir):
     missed_tests = list(python_files - set(map(lambda x: x.split()[0], ALL_SCRIPTS + NON_SCRIPTS)))
     if len(missed_tests) != 0:
         print("%sWARNING!%s The following scripts are not being run: %s. Check the test lists in test_runner.py." % (BOLD[1], BOLD[0], str(missed_tests)))
-        if os.getenv('TRAVIS') == 'true':
+        if fail_on_warn:
             # On travis this warning is an error to prevent merging incomplete commits into master
             sys.exit(1)
 
+
```

Comment:
I get that this is in the backport, but I don't like it

Reply:
This is actually PEP 8 compliant and I assume this is what the original author wanted.

Reply:
Weird, I don't like it, but understandable.